sass.py

```diff
@@ -694,12 +694,15 @@ def __new__(cls, r, g, b, a):
 SEPARATORS = frozenset((SASS_SEPARATOR_COMMA, SASS_SEPARATOR_SPACE))
 
 
-class SassList(collections.namedtuple('SassList', ('items', 'separator'))):
+class SassList(collections.namedtuple(
+        'SassList', ('items', 'separator', 'bracketed'),
+)):
 
-    def __new__(cls, items, separator):
+    def __new__(cls, items, separator, bracketed):
```

Comment:
If you meant giving any default value (`True` or `False`) is inappropriate, how about making this parameter to be `False` by default, but warn (i.e. [`DeprecationWarning`][1]) when it's not explicit?

[1]: https://docs.python.org/3/library/exceptions.html#DeprecationWarning